cmd/trillian/sqlproxy/pkg/bindings/mysql/mysql.go

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
I'm not sure we need this since we don't seem to use the bindings?

Reply:
removed